packages/node-patches/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_nodejs//:tools/defaults.bzl", "npm_package")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "node_patches",
+    srcs = [
+        "register.js",
+        "package.json",
+    ] + glob([
+        "build/src/*.js",
+        "build/src/**/*.js",
+    ]),
+)
+
+npm_package(
+    name = "npm_package",
+    srcs = [
+        "readme.md",
+        ":node_patches",
+    ],
+)
+
+#sh_binary(
```

Comment:
temporary target for development?

Reply:
yeah i can remove it. I just liked to be able to un-comment and see the files that were being exposed